### PR TITLE
feat: converted to API

### PR DESCRIPTION
**Pull request by Peikae3IWaeG**

### CODE REVIEW

File: back/generator/model/keywords.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    name: str = "RW.CLI.Run Cli"
+
+    def __init__(
+        self,
+        assign_to_variable=False,
+        variable="",
+        cmd: str = "string",
+        render_in_commandlist: bool = True,
+        target_service: any = None,
+        secret_file__kubeconfig: any = None,
```
Comment (jon-funk) [inline back/generator/model/keywords.py:116]: FYI - the `secret__` and `secret_file__` keyword args for the `Run CLI` keyword are dynamic kwargs. They produce a secret of the name at the end of the dunder name (kubeconfig in this case) with a secret that has the contents of the secret object passed in. Here's an example with the gcp CLI for comparison: https://github.com/runwhen-contrib/rw-cli-codecollection/blob/main/codebundles/gcloud-log-inspection/runbook.robot#L60

Not a problem while you're getting it working but keep it in mind for future iterations.

Comment (Peikae3IWaeG) [reply back/generator/model/keywords.py:116]: Sure, this is definitely going to be updated the way you pointed out. I'll focus on adjusting parameters support right after delivering the remaining functionalities skeletons.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,5 @@
-docker build -t robotgen .
-docker run --name robotgen --rm -p 5000:5000 robotgen
+add endpoint in docker-compose.yml
```
[the remark this answers is not in the document]
Comment (Peikae3IWaeG) [reply README.md]: Good point, I forgot to do so, as well as adding volume mont for hot reload. Both added now

File: README.md [hunk 3]
```diff
@@ -1,2 +1,5 @@
...
+docker compose up
+
+ui: port 3000
+api: port 5000
```
[the remark this answers is not in the document]
Comment (Peikae3IWaeG) [reply README.md]: added

File: front/app/components/variables.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+      .catch((error) => console.error("Error fetching API data:", error));
+  }, []);
+
+  return (
+    <div>
+      <TableContainer component={Paper}>
+        <Table sx={{ minWidth: 650 }} size="small" aria-label="a dense table">
+          <TableHead>
+            <TableRow>
+              <TableCell align="right">Name</TableCell>
```
[the remark this answers is not in the document]
Comment (Peikae3IWaeG) [reply front/app/components/variables.js:28]: I added a new component and dedicated endpoints for handling secrets. wdyt

**Pull request merged**